pcre/CMakeLists.txt

```diff
@@ -125,7 +125,7 @@ OPTION(PCRE_BUILD_PCRE16 "Build 16 bit PCRE library" OFF)
 
 OPTION(PCRE_BUILD_PCRE32 "Build 32 bit PCRE library" OFF)
 
-OPTION(PCRE_BUILD_PCRECPP "Build the PCRE C++ library (pcrecpp)." ON)
+OPTION(PCRE_BUILD_PCRECPP "Build the PCRE C++ library (pcrecpp)." OFF)
```

Comment:
For all changes to this directory: No.

This is a literal unmodified source dump from PCRE. Don't go modifying things in it.